ajax/libs/DisplayJS/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "DisplayJS",
+  "filename": "display.min.js",
+  "version": "0.3.2",
+  "description": "A simple JavaScript framework for building ambitious UIs",
+  "keywords": [
+    "Display.JS",
+    "display",
+    "DOM",
+    "manipulation",
+    "framework",
+    "javascript"
+  ],
+  "author": "Arthur Guiot",
+  "license": "MIT",
+  "homepage": "https://display.js.org",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arguiot/DisplayJS.git"
+  },
+  "npmName": "display.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
I don't know that is `display.es6.js` can work on web front-end?
If this file can't use in front-end, let's modify from `"**/*"` to `"**/!(*.es6)*"`.
Thank you.